web/main/helpers.ts

```diff
@@ -0,0 +1,13 @@
+export const actualizeThemeColor = () => {
```

Comment:
The helper is too small to move it to another file. Move it to `browser.ts`, it will be easier to analyze it.